tools/dep_updaters/update-gyp-next.sh

```diff
@@ -55,14 +55,6 @@ rm -rf "$BASE_DIR/tools/gyp"
 
 mv "$WORKSPACE/gyp" "$BASE_DIR/tools/"
 
-echo "All done!"
-echo ""
-echo "Please git add gyp-next and commit the new version:"
-echo ""
-echo "$ git add -A tools/gyp"
```

Comment:
`tools/dep_updaters/update-gyp-next.sh` does not contain `finalize_version_update` so it cannot be removed